ppa/archive/tests/test_views.py

```diff
@@ -418,13 +418,37 @@ def test_digitizedwork_listview(self):
         assert not response.context['object_list'].count()
         self.assertContains(response, 'Invalid range')
 
+        # ajax request for search results
+        response = self.client.get(url, HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+        assert response.status_code == 200
+        # should render the results list partial and single result partial
+        self.assertTemplateUsed('archive/snippets/results_list.html')
+        self.assertTemplateUsed('archive/snippest/search_result.html')
+        # shouldn't render the search form
+        self.assertNotContains(response, '<form class="ui form">', html=True)
+        # should have all the results
+        assert len(response.context['object_list']) == len(digitized_works)
```

Comment:
add basic sanity check for the data that needs to be present for the ajax form logic to update count and histogram